examples/00-load/create-polyhedron.py

```diff
@@ -0,0 +1,196 @@
+"""
+.. _polyhedron_example:
+
+Unstructured Grid with Polyhedra
+--------------------------------
+
+This example shows how to build a simple :class:`pyvista.UnstructuredGrid`
+using polyhedra. We will be using VTK types to determine which type of cells we
+are building.
+
+First, we import the required libraries.
+"""
+
+import pyvista as pv
+
+###############################################################################
+# Define Points
+# ~~~~~~~~~~~~~
+# We will mix several cells in one grid for this example. Here we create the
+# points that will define each cell.
+#
+# .. note::
+#    It is not necessary that each cell has an isolated set of points. This has
+#    been done here to create isolated cells for this example.
+
+
+quad_points = [
+    [0.0, 0.0, 0.0],  # 0
+    [0.0, 0.01, 0.0],  # 1
+    [0.01, 0.01, 0.0],  # 2
+    [0.01, 0.0, 0.0],  # 3
+]
+polygon_points = [
+    [0.02, 0.0, 0.0],  # 4
+    [0.02, 0.01, 0.0],  # 5
+    [0.03, 0.01, 0.0],  # 6
+    [0.035, 0.005, 0.0],  # 7
+    [0.03, 0.0, 0.0],  # 8
+]
+hexa_points = [
+    [0.0, 0.0, 0.02],  # 9
+    [0.0, 0.01, 0.02],  # 10
+    [0.01, 0.01, 0.02],  # 11
+    [0.01, 0.0, 0.02],  # 12
+    [0.0, 0.0, 0.03],  # 13
+    [0.0, 0.01, 0.03],  # 14
+    [0.01, 0.01, 0.03],  # 15
+    [0.01, 0.0, 0.03],  # 16
+]
+polyhedron_points = [
+    [0.02, 0.0, 0.02],  # 17
+    [0.02, 0.01, 0.02],  # 18
+    [0.03, 0.01, 0.02],  # 19
+    [0.035, 0.005, 0.02],  # 20
+    [0.03, 0.0, 0.02],  # 21
+    [0.02, 0.0, 0.03],  # 22
+    [0.02, 0.01, 0.03],  # 23
+    [0.03, 0.01, 0.03],  # 24
+    [0.035, 0.005, 0.03],  # 25
+    [0.03, 0.0, 0.03],  # 26
+]
+points = quad_points + polygon_points + hexa_points + polyhedron_points
+
+
+###############################################################################
+# Cell connectivity
+# ~~~~~~~~~~~~~~~~~
+# Connectivity describes the indices of the points to compose each cell. The
+# first item in each cell's connectivity is the number of item the cell will
+# have. For example, a quad cell is composed of point ``[0, 1, 2, 3]`` and
```

Comment:
I still think this is ambiguous unless we pluralise `points`. This currently sounds as if `[0, 1, 2, 3]` was somehow a single point. But I'm not a native.
```suggestion
# have. For example, a quad cell is composed of points ``[0, 1, 2, 3]`` and
```